shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -2166,23 +2196,20 @@ export default {
             {{ t('cluster.rke2.security.header') }}
           </h3>
           <Banner
-            v-if="isEdit && displayInvalidPspsBanner"
+            v-if="isEdit && displayInvalidPspsBanner && hasPsps"
```

Comment:
If I remember correctly, we should display the warning banner (`invalidPsps`) only if there are PSPs in the Cluster, not the `templates`.
Given this, we should remove `displayInvalidPspsBanner` from the `v-if` statement, because it's checking for the PSPs templates; as it works now, the warning message is shown only when both PSPs `and` PSPs templates are in the Cluster

Reply:
We are checking the presence of PSPs, which include the use of templates.

Good point about the code deprecation.